random_menu.rb

```diff
@@ -0,0 +1,11 @@
+my_list = [
+  "foo",
+  "bar",
+  "mar"
+]
+
+my_list.each do |item|
```

Comment:
This is an inline comment! Whoever gave feedback had something to say about this particular line of code.